packages/components/source-repositories/src/repository.spec.ts

```diff
@@ -0,0 +1,100 @@
+import { Blueprint } from '@amazon-codecatalyst/blueprints.blueprint';
+import { SourceFile } from './files/source-file';
+import { SourceRepository } from './repository';
+import { SubstitionAsset } from './static-assets';
+
+jest.mock('./files/source-file');
+
+jest.mock('./static-assets', () => ({
+  SubstitionAsset: {
+    findAll: jest.fn().mockImplementation(() => {
+      return [];
+    }),
+  },
+}));
+
+const mockSubstitutionAsset = jest.mocked(SubstitionAsset);
+const mockSourceFile = jest.mocked(SourceFile);
+
+const mockBlueprint = {
+  context: {
+    rootDir: '.',
```

Comment:
Does it matter if rootDir is something else?

Reply:
Shouldn't matter since `rootDir` isn't referenced in `copyStaticFiles`. Only mocking this since the `SourceRepository` constructor uses it to set `this.path` (which also is not referenced by `copyStaticFiles`). `this.path()` would eventually matter when writing to disk in the `File` class, but we're mocking that here so doesn't really matter.